tasks/kernel_matrix_testing/init_kmt.py

```diff
@@ -34,9 +34,6 @@ def init_kernel_matrix_testing_system(ctx: Context, lite: bool, images):
         if resp.lower().strip() != "y":
             raise Exit("Aborted by user")
 
-    reqs_file = Path(__file__).parent / "requirements.txt"
-    ctx.run(f"pip3 install -r {reqs_file.absolute()}")
-
```

Comment:
These lines are purposefully places over here, since some of the python packages depend on system packages installed by apt a [few lines above](https://github.com/DataDog/datadog-agent/blob/a49890ce48281d407dccda237bea32b1b25f7e6c/tasks/kernel_matrix_testing/init_kmt.py#L30).

The change in this PR breaks the task since `dda` fails to build dependencies due to missing packages.

Reply:
Locally, all `kmt.` tasks first dynamically install the required Python dependencies (the important one you're talking about is `libvirt-python` I assume).

- https://github.com/DataDog/datadog-agent-dev/blob/v0.5.0/src/dda/cli/inv/__init__.py#L43-L44
- https://github.com/DataDog/datadog-agent-dev/blob/v0.5.0/pyproject.toml#L169-L177

Are you saying that the existing logic that automatically installs the Python dependencies should be removed in favor of putting that here as a manual step?

Reply:
For now I added that manual logic back until we can also dynamically do the system dependencies.

Reply:
Do we need to run the `kmt.init` tasks without the `dda` prefix now?
If so, we should update the README as well.

Reply:
No because I'm going to remove that logic over there in a few minutes, and then release.

Reply:
https://github.com/DataDog/datadog-agent-dev/pull/38

Reply:
Sounds good thanks!